### PR TITLE
feat(kibbeh): update polish translation

### PR DESCRIPTION
**Pull request by PiotrekPKP**
Updated few translations & typos

### CODE REVIEW

**Comment by dk-raw** [comment]
please follow the proper pattern for commit messages *(💡TIP: you can use `yarn commit` or `yarn commit:signed` and answer the questions asked to generate a properly formatted one for you!)*

to fix your most recent commit you can run the command `git commit --amend -m "properly formatted commit message here"`

to fix your older/multiple commits please take a look at [this article](https://docs.github.com/en/github/committing-changes-to-your-project/changing-a-commit-message#amending-older-or-multiple-commit-messages)

File: kibbeh/public/locales/pl/translation.json [hunk 1]
```diff
@@ -262,7 +262,7 @@
         "makePrivate": "uczyń pokój prywatnym",
         "renamePublic": "Ustaw publiczną nazwę pokoju",
         "renamePrivate": "Ustaw prywatną nazwę pokoju",
-        "chatDisabled": "disable chat"
+        "chatDisabled": "wyłącz chat"
```
Comment (szymonjudasznakd) [inline kibbeh/public/locales/pl/translation.json]: line 154: you are using 'Polish' form `czat` but here is `chat`.
I would expect consistency

Comment (PiotrekPKP) [reply kibbeh/public/locales/pl/translation.json]: Fixed it, thanks for pointing that out!

Comment (szymonjudasznakd) [reply kibbeh/public/locales/pl/translation.json]: no problem :)

**Pull request merged**